packages/remote-config/lib/index.js

```diff
@@ -190,10 +190,9 @@ class FirebaseConfigModule extends FirebaseModule {
 
   setConfigSettings(settings) {
     const nativeSettings = {
-      // defaults to 1 minute.
-      fetchTimeoutMillis: 60000,
-      // defaults to 12 hours.
-      minimumFetchIntervalMillis: 43200000,
+      //iOS & Android expect seconds
```

Comment:
@Salakar, I changed the `nativeSettings` props because the native is slightly different to JS SDK. Android & iOS expect seconds.

Reply:
Good catch